build/golang.go

```diff
@@ -138,14 +149,45 @@ func (gm *GoModule) getPackagePathIfExists(cachePath, dependencyName, version st
 }
 
 // populateZip adds the zip file as build-info dependency
-func populateZip(packageId, zipPath string) (zipDependency *entities.Dependency, err error) {
+func populateZip(packageId, zipPath string) (zipDependency entities.Dependency, err error) {
 	// Zip file dependency for the build-info
-	zipDependency = &entities.Dependency{Id: packageId}
+	zipDependency = entities.Dependency{Id: packageId}
 	checksums, err := utils.GetFileChecksums(zipPath)
 	if err != nil {
-		return nil, err
+		return
 	}
 	zipDependency.Type = "zip"
 	zipDependency.Checksum = &entities.Checksum{Sha1: checksums.Sha1, Md5: checksums.Md5, Sha256: checksums.Sha256}
 	return
 }
+
+func populateRequestedByTree(parentDependency entities.Dependency, dependenciesMap map[string]entities.Dependency, dependenciesGraph map[string][]string) {
```

Comment:
`requestedBy` is not a tree
```suggestion
func populateRequestedByField(parentDependency entities.Dependency, dependenciesMap map[string]entities.Dependency, dependenciesGraph map[string][]string) {
```